src/main/scala/cromwell/engine/package.scala

```diff
@@ -29,6 +29,10 @@ import scala.util.{Failure, Success, Try}
  * Internally, this package is built on top of [[cromwell.binding]].
  */
 package object engine {
+
+  type Hash = String
+  type FileHasher = WdlFile => Hash
```

Comment:
Actually I just started doing it and wrapping `String`s around a `case class Hash(hash: String)` everywhere seems to just make it more annoying to instantiate and use...